R/pkg/R/functions.R

```diff
@@ -3548,7 +3548,7 @@ setMethod("row_number",
 
 #' array_contains
 #'
-#' Returns true if the array contain the value.
+#' Returns null if the array is null, true if the array contains the value, and false otherwise.
```

Comment:
for null, we need to be more careful - `null` in JVM should show up as `NA` in R.
also, should `true` be `TRUE` and `false` be `FALSE` to match R type?